public/Test-DbaWindowsLogin.ps1

```diff
@@ -59,17 +62,24 @@ function Test-DbaWindowsLogin {
 
         Tests all Domain logins excluding any that are from the testdomain
 
+    .EXAMPLE
+        PS C:\> Get-DbaLogin -SqlInstance Dev01 -Login DOMAIN\User | Test-DbaWindowsLogin
+
+        Tests only the login returned by Get-DbaLogin
+
     #>
     [CmdletBinding()]
     param (
-        [parameter(Mandatory, ValueFromPipeline)]
+        [parameter(ValueFromPipeline)]
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
-        [object[]]$Login,
-        [object[]]$ExcludeLogin,
+        [string[]]$Login,
+        [string[]]$ExcludeLogin,
```

Comment:
Breaks TEPP (ability for tab completion on the logins in an instance)

Reply:
But Get-DbaLogin also use [string] and have not problem with TEPP. I think we wanted to move from [object] to [string] whenever working on a command.

Reply:
yeah the object to string thing i think shouldnt impact tepp? it's rare that we shouldnt change it to string instead. years ago, i didnt quite understand piping and wanted to leave the params open to be various types of inputs. but have since figured out it should be controleld by InputObject isntead